src/controller.cpp

```diff
@@ -457,6 +463,17 @@ void Controller::setCooldownForAllServersInACity(const QString& countryCode,
   vpn->setCooldownForAllServersInACity(countryCode, cityCode);
 }
 
+bool Controller::hasCooldownForAllServers() {
```

Comment:
I'm concerned that this doesn't perform the same check for the entry server, which may lead to a similar bug occurring in the multihop case.

Reply:
Added a check for multi-hop.